README.md

```diff
@@ -24,6 +24,7 @@ For all the configurations you should keep an eye in 3 files:
 -   `db/.env.example` - please rename it to `.env`
 -   `frontend/.env.docker.example` - please rename it to `.env`
 -   `backend/backend/settings/example.py` - please rename it to `dev.py` and fill as required.
+-    Create an empty file at root called `.env.codecov`, this is needed in the CI env.
```

Comment:
Codacy found an issue: [[list-item-indent] Incorrect list-item indent: remove 1 space](https://app.codacy.com/manual/lgruelas/finance/pullRequest?prid=5738587)

Reply:
fixed in 6bfeeab.